test/files/pos/spec-Function1.scala

```diff
@@ -1,11 +1,15 @@
-/*                     __                                               *\
-**     ________ ___   / /  ___     Scala API                            **
-**    / __/ __// _ | / /  / _ |    (c) 2002-2013, LAMP/EPFL             **
-**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
-** /____/\___/_/ |_/____/_/ | |                                         **
-**                          |/                                          **
-\*                                                                      */
-
+/*
+ *                      __
+ *      ________ ___   / /  ___     Scala
+ *     / __/ __// _ | / /  / _ |    (c) EPFL
+ *   __\ \/ /__/ __ |/ /__/ __ |    (c) Lightbend, Inc.
+ *  /____/\___/_/ |_/____/_/ | |    https://www.scala-lang.org
+ *                           |/     Licensed under Apache License 2.0
+ *
+ *  See the NOTICE file distributed with this work for additional information
+ *  regarding copyright ownership.
+ */
+ 
```

Comment:
this doesn't seem necessary

Reply:
We used to have copyright 2012-20XX. This is a replacement for that using the wording used by Apache foundation. https://www.apache.org/legal/src-headers.html

Reply:
But this is a test file, which we don't distribute.

Reply:
Oh you mean the fact that the header is applied to `files/pos`? Some files are I think tested against source files in main source to test code generation etc, so I just manually applied headers.

Reply:
Ah, I see.